beacon-chain/core/peerdas/helpers.go

```diff
@@ -300,3 +302,13 @@ func VerifyDataColumnSidecarKZGProofs(sc *ethpb.DataColumnSidecar) (bool, error)
 	}
 	return cKzg4844.VerifyCellKZGProofBatch(ckzgComms, rowIdx, colIdx, cells, proofs)
 }
+
+// CustodySubnetCount returns the number of subnets the node should participate in for custody.
+func CustodySubnetCount() uint64 {
+	// TODO: add ability to customize custody requirement.
```

Comment:
At the moment, it is not sure we'll allow that.
Right now, it is possible to:
- Either custody the minimum amount of columns, or
- Either custody all the columns


With validator custody, the number of custody columns will depend on the ETH amount managed by attached validators.